sentry-core/src/main/java/io/sentry/core/SentryEvent.java

```diff
@@ -140,7 +141,7 @@ public void setThreads(SentryValues<SentryThread> threads) {
   }
 
   public void setExceptions(SentryValues<SentryException> exceptions) {
```

Comment:
Can we actually get rid of this method? 
`SentryValues` is an implementation detail, ideally wouldn't be exposed here. 
We have on 123 taking `List<>` and creating SentryValues